modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/tests/start_stop_client/node/Action.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.ducktest.tests.start_stop_client.node;
+
+/**
+ * Base interface.
+ */
+public interface Action {
+    /* Target action. **/
+    public long singleAction();
+
+    /* Method for publishing an internal report. **/
+    public void publishInterimReport(Report report);
```

Comment:
Do you mean `iterim` (smth between stages) or `internal`? So it seems that either comment is incorrect or method name.

Reply:
this id internal report. Ok, renamed method